tests/serverpod_test_server/test_integration/database_operations/entity_relations/entities_with_list_relations/order_by_test.dart

```diff
@@ -47,12 +116,45 @@ void main() async {
 
   group('Given ordered find query', () {
     test(
-        'when ordering on filtered count of nested many relation then result is as expected.',
+        'when ordering on filtered count of nested many relation then result order is as expected.',
+        () async {
+      var persons = await Person.db.find(
+        session,
+        // Order people by number of people in their organization ascending
+        // and then their name.
+        orderByList: [
+          db.Order(
+            column: Person.t.organization.people.count(),
+            orderDescending: false,
+          ),
+          db.Order(
+            column: Person.t.name,
+          )
+        ],
+      );
+
+      var personNames = persons.map((e) => e.name);
+      expect(personNames, [
+        'Jane',
+        'John',
+        'Tom',
+        'Alex',
+        'Annie',
+        'Isak',
+        'Lisa',
+        'Marc',
+        'Viktor',
+        'Wendy'
+      ]);
+    });
+
+    test(
+        'when ordering on filtered count of nested many relation then result order is as expected.',
         () async {
-      var customers = await Person.db.find(
+      var persons = await Person.db.find(
         session,
         // Order people by number of people in their organization with a name
-        // starting with 'a' and then their name in descending order.
+        // starting with 'a' and then their name.
```

Comment:
Suggestion: Same here, having the same test description for several tests will not be very helpful in situations where one of the test fail but the other passes.

Reply:
This should be fixed now, there are no repetition of test descriptions anymore.